tests/quantization/test_saturation_export.py

```diff
@@ -0,0 +1,213 @@
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 128)
+    assert (quantizer.level_low == 0)
+    assert (quantizer.level_high == 127)
+
+    q_config = QuantizerConfig(
+        bits=7,
+        mode=QuantizationMode.SYMMETRIC,
+        signedness_to_force=True,
+        per_channel=True,
+        is_weights=True,
+        input_shape=[3, 32, 32],
+        is_saturation_fix=True)
+
+    quantizer = SymmetricQuantizer(q_config)
+
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 64)
+    assert (quantizer.level_low == -32)
+    assert (quantizer.level_high == 31)
+
+    q_config = QuantizerConfig(
+        bits=4,
+        mode=QuantizationMode.SYMMETRIC,
+        signedness_to_force=True,
+        per_channel=False,
+        is_weights=True,
+        is_saturation_fix=True)
+
+    quantizer = SymmetricQuantizer(q_config)
+
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 8)
+    assert (quantizer.level_low == -4)
+    assert (quantizer.level_high == 3)
+
+    q_config = QuantizerConfig(
+        bits=8,
+        mode=QuantizationMode.SYMMETRIC,
+        signedness_to_force=True,
+        per_channel=False,
+        is_weights=False,
+        is_saturation_fix=True)
+
+    quantizer = SymmetricQuantizer(q_config)
+
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 128)
+    assert (quantizer.level_low == -64)
+    assert (quantizer.level_high == 63)
+
+
+def get_config_for_export_mode(should_be_onnx_standard: bool) -> NNCFConfig:
+    nncf_config = NNCFConfig()
+    nncf_config.update({
+        "input_info": {
+            "sample_size": [1, 1, 20, 20]
+        },
+        "compression": {
+            "algorithm": "quantization",
+            "export_to_onnx_standard_ops": should_be_onnx_standard
+        },
+        "target_device": "CPU"
+    })
+    return nncf_config
+
+
+class TwoConvTestModel(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.features = []
+        self.features.append(create_conv(1, 2, 2, -1, -2))
+        self.features.append(create_conv(2, 1, 3, 0, 0))
+        self.features = nn.Sequential(*self.features)
+
+    def forward(self, x):
+        return self.features(x)
+
+
+def test_hw_config_saturation_issue_set():
+    # Test CPU device in which we use saturation issue
+    model = TwoConvTestModel()
+    nncf_config = get_config_for_export_mode(True)
+
+    compressed_model, compression_ctrl = create_compressed_model_and_algo_for_test(model, nncf_config)
+
+    for conv in [module for module in compressed_model.get_nncf_wrapped_model().features if
+                 isinstance(module, NNCFConv2d)]:
+        assert (conv.pre_ops._modules['0'].op.is_saturation_fix == True)
+        assert (conv.pre_ops._modules['0'].op.levels == 128)
+        assert (conv.pre_ops._modules['0'].op.level_low == -64)
+        assert (conv.pre_ops._modules['0'].op.level_high == 63)
```

Comment:
Use controller's fields to access weight quantizer information instead, like you did below.

Reply:
Done

Reply:
Done